snooty/parser.py

```diff
@@ -424,6 +436,54 @@ def dispatch_departure(self, node: docutils.nodes.Node) -> None:
                 target.children = [identifier]
                 item.term.append(target)
 
+    def handle_tabset(self, node: n.Directive, line: int) -> None:
+        tabset = node.options["tabset"]
+        # retrieve dictionary associated with this specific tabset
+        tab_definitions_list = specparser.SPEC.tabs[tabset]
+        tabid_list: List[str] = []
+
+        for idx, child in enumerate(node.children):
+            if not isinstance(child, n.Directive):
```

Comment:
We can probably get more specific here; we want to ensure each child is a directive named `tab`, right?